### PR TITLE
#167450752 create article card

### PR DESCRIPTION
**Pull request by akinyeleolat**
#### What does this PR do?

This PR create an article card component

#### Description of task to be completed?

- setup container for the atoms
-Implement article card
-Implement icon row for rating
-Update the style guide
-Write unit test

#### How should this be manually tested?
$ git clone https://github.com/andela/king-kong-ah-frontend.git
$ npm install
$ npm test

#### Any background context you want to provide?

You would need npm to run this application

#### What are the relevant pivotal tracker stories?

PT Story link - #167450752

#### Screenshots (if appropriate)

Screenshot 2019-07-24 at 9 06 24 PM
Screenshot 2019-07-24 at 9 06 45 PM

### CODE REVIEW

**Review by mosinmiloluwa01 — APPROVED** [review]

File: src/components/UI/atoms/Text/Text.jsx [hunk 1]
```diff
@@ -83,9 +86,11 @@ Text.Container = styled.span`
     display,
     textAlign,
     color,
+    lineHeight,
     theme,
   }) => `
     text-align: ${textAlign};
+    line-height: ${theme.lineHeights[lineHeight]};
```
Comment (EmmanuelSage) [inline src/components/UI/atoms/Text/Text.jsx:102]: good job Tosin,

I think you could give the line height a default prop, as the line-height would be undefined everywhere the text component has been used without passing the line-height prop.

Thank you.

File: src/components/UI/molecules/StarRating/StarRating.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      {renderRating()}
+    </FlexContainer>
+  );
+
+  function renderRating() {
+    const iconHolder = [];
+    iconHolder.length = 5;
+    iconHolder.fill('starEmpty');
+    let i;
+    for (i = 0; i < rating; i += 1) {
```
Comment (benfluleck) [inline src/components/UI/molecules/StarRating/StarRating.jsx]: use es6 logic for this

File: src/components/UI/molecules/ArticleCard/ArticleCard.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        width="mds"
+      />
+    );
+  }
+
+  function renderImage() {
+    return (
+      <Image
+        imageUrl={imageUrl}
+        width='288px'
```
Comment (benfluleck) [inline src/components/UI/molecules/ArticleCard/ArticleCard.jsx:49]: Does this width take into account all cards in the codebase ?

Comment (akinyeleolat) [reply src/components/UI/molecules/ArticleCard/ArticleCard.jsx:49]: yeah

**Review by benfluleck — APPROVED** [review]

**Pull request merged**